invenio_communities/cache/redis.py

```diff
@@ -0,0 +1,54 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of Invenio.
+# Copyright (C) 2023 CERN.
+#
+# Invenio is free software; you can redistribute it and/or modify it
+# under the terms of the MIT License; see LICENSE file for more details.
+
+"""Implements a Redis cache."""
+from cachelib import RedisCache
+from flask import current_app
+from redis import StrictRedis
+
+from invenio_communities.cache.cache import IdentityCache
+
+
+class IdentityRedisCache(IdentityCache):
+    """Redis image cache."""
+
+    def __init__(self, app=None):
+        """Initialize the cache."""
+        super(IdentityRedisCache, self).__init__(app=app)
+        app = app or current_app
+        redis_url = app.config["COMMUNITIES_IDENTITIES_CACHE_REDIS_URL"]
+        prefix = app.config.get("COMMUNITIES_IDENTITIES_CACHE_REDIS_PREFIX", "identity")
+        self.cache = RedisCache(host=StrictRedis.from_url(redis_url), key_prefix=prefix)
```

Comment:
Is it, in the end, necessary to bypass the cache abstraction and go directly to Redis? Is it because we discussed separating in different cache databases, so that when it is cleaned, only that db is touched?